src/com/google/enterprise/gsafeed/FeedHelper.java

```diff
@@ -82,6 +85,33 @@ public void fatalError(SAXParseException exception)
           throw exception;
         }
       };
+    this.validationEventHandler = new ValidationEventHandler() {
+        public boolean handleEvent(ValidationEvent event) {
+          boolean shouldContinue;
+          String severity;
+          switch (event.getSeverity()) {
+            case ValidationEvent.WARNING:
+              severity = "WARNING";
+              shouldContinue = true;
+              break;
+            case ValidationEvent.ERROR:
+              severity = "ERROR";
+              shouldContinue = false;
+              break;
+            case ValidationEvent.FATAL_ERROR:
+              severity = "FATAL_ERROR";
+              shouldContinue = false;
+              break;
+            default:
+              throw new IllegalArgumentException(
+                  "Unknown severity " + event.getSeverity());
+          }
+          // TODO(aptls): find out if we want logging, System.out, ...
```

Comment:
A TODO seemed OK, but I had a lean toward logging at INFO or higher, which is written to the console by default, but configurable, so maybe the best of both worlds.